local-interchaintest/examples/ibc_transfer_juno_ntrn.rs

```diff
@@ -0,0 +1,240 @@
+            .get_request_builder(JUNO_CHAIN_NAME),
+        DEFAULT_KEY,
+        &input_account,
+        &[BankCoin {
+            denom: JUNO_CHAIN_DENOM.to_string(),
+            amount: 1_000_000_000_000u128.into(),
+        }],
+        &BankCoin {
+            denom: JUNO_CHAIN_DENOM.to_string(),
+            amount: cosmwasm_std_old::Uint128::new(5000),
+        },
+    )
+    .unwrap();
+    std::thread::sleep(std::time::Duration::from_secs(3));
+
+    let start_input_balance = bank::get_balance(
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(JUNO_CHAIN_NAME),
+        &input_account,
+    )
+    .iter()
+    .find(|bal| bal.denom == ibc_denom)
+    .map_or(0, |bal| bal.amount.u128());
+    info!("Start input balance: {:?}", start_input_balance);
+
+    // We need a normal account on Neutron to be the output account for the IBC transfer service
+    let output_account = test_ctx.get_chain(NEUTRON_CHAIN_NAME).admin_addr.clone();
+    info!("Output account: {:?}", output_account);
+
+    let start_output_balance = bank::get_balance(
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(NEUTRON_CHAIN_NAME),
+        &output_account,
+    )
+    .iter()
+    .find(|bal| bal.denom == NTRN_DENOM)
+    .map_or(0, |bal| bal.amount.u128());
+    info!("Start output balance: {:?}", start_output_balance);
+
+    info!("Prepare the IBC transfer service contract");
+    let ibc_transfer_svc_contract_path = format!(
+        "{}/artifacts/valence_ibc_transfer_service.wasm",
+        current_dir.display()
+    );
+
+    let mut uploader = test_ctx.build_tx_upload_contracts();
+    uploader
+        .with_chain_name(JUNO_CHAIN_NAME)
+        .send_single_contract(&ibc_transfer_svc_contract_path)?;
+
+    // Get the code id
+    let code_id_ibc_transfer_svc = *test_ctx
+        .get_chain(JUNO_CHAIN_NAME)
+        .contract_codes
+        .get("valence_ibc_transfer_service")
+        .unwrap();
+
+    info!("Creating IBC transfer service contract");
+    let transfer_amount = 1_000_000_000u128;
+    let juno_ntrn_path = &(JUNO_CHAIN_NAME.to_string(), NEUTRON_CHAIN_NAME.to_string());
+    let ibc_transfer_instantiate_msg = valence_service_utils::msg::InstantiateMsg::<ServiceConfig> {
+        owner: JUNO_CHAIN_ADMIN_ADDR.to_string(),
+        processor: JUNO_CHAIN_ADMIN_ADDR.to_string(),
+        config: ServiceConfig {
+            input_addr: ServiceAccountType::Addr(input_account.clone()),
+            output_addr: output_account.clone(),
+            denom: UncheckedDenom::Native(ibc_denom.to_string()),
+            amount: transfer_amount.into(),
+            memo: "".to_owned(),
+            remote_chain_info: valence_ibc_transfer_service::msg::RemoteChainInfo {
+                channel_id: test_ctx
+                    .transfer_channel_ids
+                    .get(juno_ntrn_path)
+                    .unwrap()
+                    .clone(),
```

Comment:
nit: we have this syntax if you prefer to use this one:
```rust
test_ctx
                .get_transfer_channels()
                .src(JUNO_CHAIN_NAME)
                .dest(NEUTRON_CHAIN_NAME)
                .get()
```

which doesn't require unwrapping, cloning or defining the path